recipes-connectivity/pc-ble-driver/pc-ble-driver_4.1.4.bb

```diff
@@ -20,22 +19,14 @@
 # * nrftool?
 #     ** it is available from PIP repo, but required only for HEX build
 
-
-# Licenses digged out by recipetool, remove comments when clarified.
+# LICENSE - Nordic-5-Clause license is used.
```

Comment:
LICENSE = "5-Clause Nordic License"

Reply:
From the Nordic ticket:

"The license is a proprietary license. It could be called just that, "Proprietary license", or if a more descriptive name is asked for, "Nordic Semiconductor 5-clause license" (Nordic has 3 different variants: 3-, 4- and 5-clause licenses)."

I suppose we should add that Semiconductor there.

The naming convention however seems to be that the "name" is 1st, then the number of clauses. See SPDX page:

https://spdx.org/licenses/BSD-4-Clause.html

for example.